invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/api/api.js

```diff
@@ -0,0 +1,39 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { http } from "react-invenio-forms";
+
+export class GrantAccessApi {
```

Comment:
@zzacharo @kpsherva 
In general, there is quite a mess with the organization of the APIs objects. They should be part of a component/module (e.g. administration, or landing page), but they should be organized in a similar way as the backend services.

For example:
```
js/invenio_app_rdm/api
  /records
    /records.js
    /grants.js (maybe inside an `access` module or `record.js`?)
```
I think that we have done a decent job in ILS: https://github.com/inveniosoftware/react-invenio-app-ils/tree/master/src/lib/api

Unfortunately, I see many `api` modules that should be extracted and moved...

Reply:
I agree, it would be better to organize similarly, I could volunteer to refactor at some point ;)